packages/core/src/components/Alert/Alert.jsx

```diff
@@ -29,11 +46,13 @@ export class Alert extends React.PureComponent {
 }
 
 Alert.propTypes = {
+  ariaLabelledBy: PropTypes.string,
```

Comment:
You can remove this prop if you choose to use `this.headingId` for the ID and aria-labelledby references.